apps-client/backoffice/app/routes/_app/feed.announcement.$announcementId/route.tsx

```diff
@@ -225,11 +234,7 @@ function AnnouncementDetailPage() {
                 <Calendar size={16} />
                 <span>วันที่สร้าง:</span>
               </div>
-              <div>
-                {new Date(query.data.createdAt).toLocaleDateString('th', {
-                  dateStyle: 'short',
-                })}
-              </div>
+              <div>{formatDisplayDate(new Date(query.data.createdAt))}</div>
```

Comment:
to me i prefer calling `dayjs(query.data.createdAt).format('DD/MM/BBBB')`
myself in case we want to edit the format we can edit inline
for example `.format('DD/MM/BBBB HH:mm')`